### PR TITLE
refactor test_languages

### PR DESCRIPTION
**Pull request by dpprdan**
I wanted to add some more examples to test_languages.R but found the test layout a bit hard to follow. I therefore propose these changes:

1. define dates to be tested only once per test and use these for both functions
2. define dates pre- and post-conversion side-by-side

I am using named vectors for this. IMHO this is a lot easier to follow, let me know what you think.

### CODE REVIEW

**Comment by nathansam** [comment]
This looks fantastic! Thank you! This is much cleaner and clearer than the previous version. I will modify the other tests in the future to follow the same format. Just as a heads up, `datefixR` uses [styler](https://github.com/r-lib/styler) to format R code, so the pretty spacing might not be maintained once the GH action is run for it.

**Comment by nathansam** [comment]
Thanks for making take a closer look at the presentation of the tests. I will do a proper refactor of the tests tonight.

**Comment by dpprdan** [comment]
> Just as a heads up, `datefixR` uses [styler](https://github.com/r-lib/styler) to format R code, so the pretty spacing might not be maintained once the GH action is run for it.

styler might [detect the alignment](https://styler.r-lib.org/articles/detect-alignment.html), if we're lucky. 😄
